templates/commandline.tmpl

```diff
@@ -59,12 +59,12 @@ echo "{{ .ClusterCA }}" \ > ca-{{ .ClusterName }}.pem
 kubectl config set-cluster {{ .ClusterName }} --server={{ .APIServerURL }} --certificate-authority=ca-{{ .ClusterName }}.pem --embed-certs
 kubectl config set-credentials {{ .Email }}  \
     --auth-provider=oidc  \
-    --auth-provider-arg='idp-issuer-url={{ .IssuerURL }}'  \
-    --auth-provider-arg='client-id={{ .ClientID }}'  \
-    --auth-provider-arg='client-secret={{ .ClientSecret }}' \
-    --auth-provider-arg='refresh-token={{ .RefreshToken }}' \
-    --auth-provider-arg='id-token={{ .IDToken }}'
-kubectl config set-context {{ .ClusterName }} --cluster={{ .ClusterName }} --user={{ .Email }}
+    --auth-provider-arg=idp-issuer-url={{ .IssuerURL }}  \
+    --auth-provider-arg=client-id={{ .ClientID }}  \
+    --auth-provider-arg=client-secret={{ .ClientSecret }} \
+    --auth-provider-arg=refresh-token={{ .RefreshToken }} \
+    --auth-provider-arg=id-token={{ .IDToken }}
+kubectl config set-context {{ .ClusterName }} --cluster={{ .ClusterName }} --user={{ .Email }} --namespace={{ .Namespace }}
```

Comment:
If namespace claim is not set, wouldn't this give you an invalid config? Or does kubectl ignore if the value is empty?